Source/Fuse.Elements/Drawing/ImageFill.uno

```diff
@@ -110,7 +170,8 @@ namespace Fuse.Drawing
 			public float2 Origin, Size;
 			public float4 UVClip;
 			public Texture2D Texture;
-			public float2 TexCoordBias1, TexCoordBias2, TexCoordScale1, TexCoordScale2;
+			public float2 TexCoordBias1, TexCoordBias2, TexCoordScale1, TexCoordScale2, TexCoordOffset, ScaleOrigin;
+			public float ScaleFactor;
```

Comment:
It's not clear to me what TexCoordBias1/2 and Scale1/2 do. 

As for reasoning, the primary reason you'd want to use ImageFill for anything at all is to get properly masked non-rectangular images. Animations are common for Image but cannot easily be done on elements masked through ImageFills. It seems to me, at least, to be a pretty reasonable _need_ rather than a weird edge case.